src/main/java/com/questhelper/helpers/quests/perilousmoon/PerilousMoon.java

```diff
@@ -0,0 +1,702 @@
+		useTalismans = new DetailedQuestStep(this,
+			"Use the talismans to find the correct location. More details in the sidebar.", infusedWaterTalisman, infusedEarthTalisman);
+		useTalismansSidebar.addSubSteps(useTalismans);
+		talkToEyatlalli = new NpcStep(this, NpcID.EYATLALLI, "Talk to Eyatlalli.");
+		talkToJessamineAfterEyatlalli = new NpcStep(this, NpcID.JESSAMINE_12865, new WorldPoint(1440, 9639, 1),
+			"Talk to Jessamine near the Monolith again.");
+//
+		enterStreamboundAgain = new ObjectStep(this, ObjectID.ENTRANCE_51376, new WorldPoint(1458, 9650, 1),
+			"Enter the north-east entrance to the streambound cavern.");
+		collectGrub = new ObjectStep(this, ObjectID.GRUBBY_SAPLING, new WorldPoint(1517, 9690, 0),
+			"Collect a grub from the grubby sapling east of the camp.");
+		getHerbloreSupplies = new ObjectStep(this, ObjectID.SUPPLY_CRATES, new WorldPoint(1511, 9695, 0),
+			"Take herblore supplies from the supply crates in the camp. You only need the pestle and mortar.");
+		getHerbloreSupplies.addDialogStep("Take herblore supplies.");
+		useGrubOnPestle = new DetailedQuestStep(this, "Use a pestle and mortar on the moonlight grub.", pestleAndMortar.highlighted(), moonlightGrub.highlighted());
+		getFishingSupplies = new ObjectStep(this, ObjectID.SUPPLY_CRATES, new WorldPoint(1511, 9695, 0),
+			"Take fishing supplies from the supply crates in the camp. You only need the big fishing net and knife.");
+		getFishingSupplies.addDialogStep("Take fishing supplies.");
+		fishBream = new ObjectStep(this, NullObjectID.NULL_51367, new WorldPoint(1521, 9689, 0),
+			"Fish a bream from the stream east of the camp.", bigFishingNet);
+		useKnifeOnBream = new DetailedQuestStep(this, "Use a knife on the raw bream.", knife.highlighted(), bream.highlighted());
+		getHunterSupplies = new ObjectStep(this, ObjectID.SUPPLY_CRATES, new WorldPoint(1511, 9695, 0),
+			"Take hunter supplies from the supply crates in the camp. You only need the rope, and the knife from the fishing supplies.");
+		getHunterSupplies.addDialogStep("Take hunting supplies.");
+		enterSmallEntrance = new ObjectStep(this, ObjectID.ENTRANCE_51378, new WorldPoint(1522, 9720, 0),
+			"Enter the entrance north of the camp.", rope);
+		setTrap = new ObjectStep(this, ObjectID.ROCK_51359, new WorldPoint(1377, 9682, 0),
+			"Set up a trap by clicking two rocks near one another.", true, rope);
+		rustleBush = new ObjectStep(this, ObjectID.BUSH_51358, new WorldPoint(1380, 9684, 0), "Rustle the nearby bush to trap a lizard.");
+		pickUpMossLizard = new ItemStep(this, "Pick up the raw moss lizard.", rawMossLizard);
+		useKnifeOnLizard = new DetailedQuestStep(this, "Use a knife on the raw moss lizard.", knife.highlighted(), rawMossLizard.highlighted());
+		leaveEarthboundWithLizard = new ObjectStep(this, ObjectID.ENTRANCE_51375, new WorldPoint(1404, 9716, 0),
+		"Leave the area back to the antechamber via the north-east entrance.");
```

Comment:
Fix object id, coords, and text to return from lizard place
before:
![image](https://github.com/Zoinkwiz/quest-helper/assets/962989/09f4fd36-7521-4598-9a7d-826ea0fdd5dc)

after:
![image](https://github.com/Zoinkwiz/quest-helper/assets/962989/0304b817-f9f7-4797-a110-136e34ed50ce)

```suggestion
		leaveEarthboundWithLizard = new ObjectStep(this, ObjectID.ENTRANCE_51378, new WorldPoint(1389, 9674, 0),
		"Leave the area back to the antechamber via the south-east entrance.");
```